server/routes/discover.ts

```diff
@@ -80,8 +85,11 @@ discoverRoutes.get('/movies', async (req, res, next) => {
   const tmdb = createTmdbWithRegionLanguage(req.user);
 
   try {
-    const query = QueryFilterOptions.parse(req.query);
+    const fullQuery = QueryFilterOptions.parse(req.query);
+    // eslint-disable-next-line @typescript-eslint/no-unused-vars
+    const { certificationMode, ...query } = fullQuery;
```

Comment:
The ESLint warning seems legit. Why destructure `certificationMode` from the query if you're not using it?